src/main/java/com/mongodb/kafka/connect/util/jmx/SourceTaskStatistics.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.kafka.connect.util.jmx;
+
+import java.util.concurrent.TimeUnit;
+
+public class SourceTaskStatistics implements SourceTaskStatisticsMBean {
+
+  // Timings
+  private long taskTimeNanos;
```

Comment:
I'm not sure what is considered best practice for JMX, but it occurs to me that reads of all of these values will be happening from a different thread than the writes, so some synchronization seems to be required.

Reply:
I guess there's no harm in making these `AtomicLong`. (Though I can't think of a case where this would have a notable effect. JMX doesn't query the properties atomically, so there's already no guarantee that they would correspond to each other at a queried point in time.)

Reply:
This is one of those times when I miss Valentin. :)

I think there are a couple of reasons to use AtomicLong here.  

One is that reads of longs are not guaranteed by the JVM to be atomic.  They can be sheared (not sure that's the right term) so that you can read a value that never actually existed 

The second is that the JVM does not guarantee that an observing thread will ever see updates to these values without some sort of synchronization.  They are allowed to remain in per-core caches indefinitely.